### ISSUE TITLE
Develop customizable UI API

### ISSUE THREAD

**Issue by flackr** [flackr/circ#384]
We should have an easy way to customize the CIRC interface. I'm not sure if relying on custom CSS is sufficient - but perhaps combined with some script which assembles the interface it is. This issue tracks developing an initial API and a theme based on it.

**Comment by flarefluffsune** [comment]
Would this allow the implementation of +vhoaq icon's for usernames. My channel runs on moderated mode and it's difficult to know if someone does or does not have voice without them messaging me.

**Comment by flackr** [comment]
Yes, adding an icon next to users, or otherwise customizing the message presentation will be possible.
